File: examples/http-url-connection/app/src/main/java/com/trackingplan/examples/urlconnection/ExampleApplication.java
package com.trackingplan.examples.urlconnection;

import android.app.Application;

import com.trackingplan.client.sdk.Trackingplan;

public class ExampleApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        System.out.println("ExampleApplication::onCreate");
        Trackingplan.init("YOUR_TP_ID")
                // .environment("PRODUCTION")
                .enableDebug()
                .dryRun()
                .start(this);
        // Trackingplan.stop();
    }
}

File: examples/okhttp3-kotlin/app/src/main/java/com/trackingplan/examples/okhttp/ExampleApplication.kt
package com.trackingplan.examples.okhttp

import android.app.Application
import com.trackingplan.client.sdk.Trackingplan

class ExampleApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        Trackingplan.init("YOUR_TP_ID")
            .enableDebug()
            .dryRun()
            .start(this)
    }
}